stream/server/src/main/java/org/apache/bookkeeper/stream/server/service/RegistrationStateService.java

```diff
@@ -95,6 +95,10 @@ protected void doStart() {
 
     @Override
     protected void doStop() {
+        if (null == stateManager) {
+            log.warn("State Manager is null, no need to stop it.");
```

Comment:
A warn message for me indicates a potential action from the user. In this case, it is just information that the state manager is null, so no action is required, correct? If so, then it is an info message at best, if not debug.

Reply:
yes